docs/meetings/2023/README.md

```diff
@@ -6,3 +6,4 @@ Resources from the KYOSS monthly meetings in 2023.
 * [February meeting](02-February)
 * [March meeting](03-March)
 * [April meeting](04-April)
+* We had meetings in May through October, but neglected to archive the materials here
```

Comment:
Would this be better line-by-line (i.e. "[June meeting - notes not found]?

Reply:
Thanks @Alchemyst19 -- I made the change.